tutorials/solution_strategies.ipynb

```diff
@@ -0,0 +1,126 @@
+{
```

Comment:
<p>Thanks for the cleanup, readability is much improved. I see you still use the same parameter dictionary for physics and numerics, but I suggest we discuss this in more general terms in a different context. So, I have no requests for further changes.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pmgbergen/porepy/pull/1208/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
Yes, that was my thinking as well.